articles/becoming-a-flutter-developer/index.md

```diff
@@ -0,0 +1,179 @@
+
+
+### Getting a job as a Flutter Developer
+
+1. #### Job Description Example 
+
+Here is an example of a detailed job description for a Flutter Developer.
+[Flutter Developer Job Description.](https://angel.co/company/monke/jobs/913854-flutter-app-developer-job-description?__cf_chl_captcha_tk__=9ab1ad7123365882306886fecbdf402053e1412b-1608831940-0-AXup_-VE35sp8me0EcIi4aB0uXcivnhClX8oYDo4NyqvlPo0lmNxt8jg6UbGsfcD57GmK-QiWcDXBPl_dXGLAX2y_RA1get_KFy0hubkIIFu4osFkdfze8zY-n5Pct_aEbZE1oG7zSAxhpPkTQvsRP9PZ9Rb_8eka9EV74xY9fE2dTvx-panKUA3JgD9P4r_nTelw5qc8Ks37-9fHxGE_q02r0e0G0N92-9zSh83rGSbr_4yC2LMI2zs8DSaaY7cuG2BR7VJrYG7nhcEHDuLVxwe5nr7TfZNlVtSnoi8LRlwNAqHeRbjFfEsm3O5LimtySi64G6PER9TOzSZj7In5UCZ8fer4sNxnLRX6aprYxjh-yBUe-mx1-ajRbM1JDsUHjFYZlYMEHKAU34cv55hLvrxsUaHtbxfWCAW9kIMLHbgTwhQxjz0iXFLsn87NzCgnsgpJWUp9LDRlkkuo8FLYxHVTirvclX3U2jHMIHWXPYCDrhgFIcTJNwjYxFOwzGi1emQobL-tdDRVIapLuDp3REw4wNrcoGaQsQLcWFJyq7dYC3qapmzlk-bECgaQYcPP3iTphpZeiLtE4w_78A-KuTYRCC3cOK34unH5ZKWdcN0_QkXvgHIR0foy5SuzMNrwl-ojJALklwnknEiGAoL6R5Zd3xPM7GtG32DFeH-lgrB)
+
+2. #### Flutter Developer Interview Questions Example
+
+Technical questions a Flutter Developer gets during an interview are mostly categorised into two:
+
+1. **Written Questions.**
+
+They mostly test your knowledge on `Dart Programming Language`. Its data structures and algorithms, control flows and general knowledge in the programming language.
+
+2. **Verbal Questions.**
+
+These questions will test your knowledge on Flutter framework. They may also include questions on Dart Programming Language.
+
+
+You can have a look at these resources to see more interview questions:
+
+- [Flutter Interview Questions and Answers.](https://www.raywenderlich.com/10971345-flutter-interview-questions-and-answers)
+
+- [Common Flutter Interview Questions and Answers.](https://www.fullstack.cafe/blog/flutter-interview-questions)
+
+- [Top Flutter Interview Questions.](https://www.javatpoint.com/flutter-interview-questions)
+
+### Further Reading
+
+To get a better grip on the topic, take a look at these resources:
+
+- To start on [Dart programming language](https://www.udemy.com/course/dart-beginners-course/?couponCode=7CA93B47A390A81961E3).
+
+- To start on [Flutter Framework](https://flutter.dev/).
+
+- To learn more about `DevTools`
+
+<iframe width="478" height="269" src="https://www.youtube.com/embed/nq43mP7hjAE" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+
+
+### Conclusions
+To become a flutter developer is simpler than becoming other professions in the software engineering field. If one knows development using native languages, it will be easier. Dart is a friendly language and Flutter an easy-to-understand framework. Becoming a Flutter Developer is a very interesting, fun and the field is still growing as the industry gets more comfortable with Flutter framework.
```

Comment:
Can you say `becoming a Flutter developer is simple`. instead of comparing it with other professions. I am sure some readers will disagree with the original sentence.